haproxy/metadata.csv

```diff
@@ -185,8 +185,11 @@ haproxy.process.unstoppable.jobs,gauge,,,,Current number of active jobs that can
 haproxy.process.uptime.seconds,gauge,,,,How long ago this worker process was started (>= 2.4).,0,haproxy,
 haproxy.server.bytes.in.total,count,,byte,,Current total of incoming bytes.,0,haproxy,
 haproxy.server.bytes.out.total,count,,byte,,Current total of outgoing bytes.,0,haproxy,
+haproxy.server.check.code,gauge,,,,"layer5-7 code, if available of the last health check. (>= 2.4)",0,haproxy,
```

Comment:
this is not correct, I fixed that in https://github.com/DataDog/integrations-core/pull/8631